projects/pedal/src/main.c

```diff
@@ -0,0 +1,54 @@
+#include "ads1015.h"
+#include "can.h"
+#include "event_queue.h"
+#include "fsm.h"
+#include "gpio.h"
+#include "gpio_it.h"
+#include "interrupt.h"
+#include "log.h"
+
+
+#include "throttle.h" 
+#include "drive_fsm.h"
+
+
+static Fsm fsm; 
+static Ads1015Storage ads1015_storage; 
+static ThrottleStorage throttle_storage; 
```

Comment:
Seems copy-pasted, do any of your modules actually use `ads1015` at this point? If not, remove it. If yes, make sure you make a mock for `ads1015` in `x86` since you're not testing with hardware right now.